src/ExportPro.Common/ExportPro.Common.Shared/Library/ApiServiceResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Net;
+
+namespace ExportPro.Common.Shared.Library
+{
+    public class ApiServiceResponse
+    {
+        public string? Message { get; set; }
+        public string DetailsMessage { get; set; }
+        public HttpStatusCode ApiState { get; set; }
+        public string? ErrorCode { get; set; }
+        public List<string> ValidationErrors { get; set; }
```

Comment:
maybe we can use Message, list of Message instead of using ValidationErrors

public List<string>? Message { get; set; }